rotkehlchen/tests/unit/accounting/evm/test_balancer.py

```diff
@@ -31,20 +29,6 @@
 
 DEPOSIT_ENTRIES = [
     EvmEvent(
-        tx_hash=EVM_HASH,
```

Comment:
why remove 1 entire event here?

Reply:
As commented on discord the test didn't have the rules for accounting the gas event and was skipping it. I tried first to add the processed event but it complicated the logic and since this test just tests the logic for the curve decoded events I saw it easier to delete it.